migrations/versions/25a80caf27de_add_sepa.py

```diff
@@ -0,0 +1,40 @@
+"""Add SEPA
+
+Revision ID: 25a80caf27de
+Revises: 94cbf12be25a
+Create Date: 2023-01-22 02:46:56.783644
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '25a80caf27de'
+down_revision = '94cbf12be25a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('sepa_mandate',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('person_id', sa.Integer(), nullable=True),
+    sa.Column('creation_date', sa.Date(), nullable=False),
+    sa.Column('adress', sa.UnicodeText(), nullable=False),
+    sa.Column('iban', sa.String(), nullable=False),
+    sa.Column('bank', sa.String(), nullable=False),
+    sa.Column('given', sa.Boolean(), nullable=False),
+    sa.Column('default', sa.Boolean(), nullable=False),
+    sa.Column('value', sa.Integer(), nullable=True),
+    sa.ForeignKeyConstraint(['person_id'], ['person.id'], name=op.f('fk_sepa_mandate_person_id_person')),
+    sa.PrimaryKeyConstraint('id', name=op.f('pk_sepa_mandate'))
+    )
+    # ### end Alembic commands ###
+
+
+def downgrade():
```

Comment:
This should not be allowed ;)

Reply:
It is possible in alembic to not implement the downgrade. Since this might remove data we need to keep, maybe this would be a point where we don't allow the downgrade.